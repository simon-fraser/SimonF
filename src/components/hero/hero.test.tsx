import { render } from '@testing-library/react';
import { it, describe, expect } from 'vitest';
import React from 'react';
import Hero from './hero'

describe('components/hero', () => {
  it('will render correctly', () => {
    const {asFragment} = render(<Hero />)

    expect(asFragment()).toMatchSnapshot()
  })
})