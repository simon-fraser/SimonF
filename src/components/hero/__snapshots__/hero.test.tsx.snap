// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components/hero > will render correctly 1`] = `
<DocumentFragment>
  <article
    class="absolute bg-black bg-opacity-30 flex h-screen inset justify-center w-screen z-10"
  >
    <div
      class="text-center flex-col flex h-full items-center justify-center prose text-white drop-shadow-md"
    >
      <h1
        class="text-white text-6xl mb-0"
      >
        Hi iâ€™m Simon Fraser
      </h1>
      <p>
        A software engineering manager, developer and enterprise 
        <span
          class="underline"
          title="and TypeScript"
        >
          JavaScript
        </span>
         application specalist.
      </p>
    </div>
  </article>
</DocumentFragment>
`;
